pages/1.12/release-notes/1.12.4/index.md

```diff
@@ -0,0 +1,217 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.12.4
+title: Release Notes for 1.12.4
+menuWeight: 5
+excerpt: Release notes for DC/OS 1.12.4
+---
+
+DC/OS Version 1.12.4 was released on June 21, 2019.
+
+[button color="light" href="https://downloads.dcos.io/dcos/stable/1.12.4/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="purple" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise [/button]
+
+DC/OS 1.12.4 includes the following components:
+- Apache Mesos 1.7.3 [change log](https://github.com/apache/mesos/blob/5e234c8d8edc1bb73ba557f5774c609fa460c9e7/CHANGELOG).
+- Marathon 1.7.203 [change log](https://github.com/mesosphere/marathon/blob/b26a8b310561934071c5f347ee5e184a3279cabd/changelog.md).
+- Metronome 0.6.18 [change log](https://github.com/dcos/metronome/blob/b4016b01a349b15df25970877bd62521a49d0cc9/changelog.md).
+
+<!-- <p class="message--note"><strong>NOTE: </strong>DC/OS 1.12.1 release supports new CoreOS and Docker versions as listed in the <a href="../../../version-policy">compatibility matrix</a>.</p> -->
+
+# Release summary
+
+DC/OS is a distributed operating system that enables you to manage resources, application deployment, data services, networking, and security in an on-premise, cloud, or hybrid cluster environment. 
+
+# Issues fixed in DC/OS 1.12.4
+The issues that have been fixed in DC/OS 1.12.4 are grouped by feature, functional area, or component. Most change descriptions include one or more issue tracking identifiers for reference.
+
+## Admin Router
+- Changes the maximum size allowed for uploads to a service through Admin Router (DCOS-52768).
+
+    This release increases the maximum size allowed for uploading packages from 1GB to 16GB. This change enables you to upload larger packages to a registry service without timing out the upload connection.
+
+
+
+## Command-line interface (CLI)
+- Provides improved clean-up operations to eliminate issues caused by unused volumes for removed comtainers (DCOS_OSS-1502). 
```

Comment:
I just want to clarify that this is not about CLI. `dcos-docker-gc` is a systemd-controlled service that runs hourly on every cluster node and cleans up docker stuff there.

Reply:
Moved under storage and added the extra info from above.